### ISSUE TITLE
Requires that ports be numbered

### ISSUE THREAD

**Issue by broonie** [pdudaemon/pdudaemon#144]
Older versions of pdudaemon (the version in Debian oldstable at least) accepted string values as ports. This was useful for my system where I have devices controlled via a command line program which don't meaningfully have numbered ports, they have scripts that get run to power them on and off, so I assigned meaningful names to them to make the system more understandable. This really feels like a regression in functionality.

**Comment by sjoerdsimons** [comment]
non-numeric port identifiers should work again since b30c33274cf341cf3c9995cd3384c1fd1207ffec - Can you confirm what version your running and at a guess you're using the localcmd driver for this setup ?

**Comment by obbardc** [comment]
@broonie i have updated pdudaemon in debian & backports , can you confirm it works OK for you now ?

names for ports works fine in HTTP mode (albet I didn't check the localcmd driver); in TCP mode for some reason only accepts integers. Didn't look into it as I don't use TCP mode.
